app/src/main/java/com/google/android/samples/socialite/ui/player/preloadmanager/PreloadManagerWrapper.kt

```diff
@@ -57,6 +57,10 @@ private constructor(
 
     /** Builds a preload manager instance with default parameters. Preload manager should use the same looper and load control as the player */
     companion object {
+
+        // Thread on which the preload manager is running
+        private lateinit var playbackAndPreloadThread: Looper
```

Comment:
`Looper` is a different concept than `Thread`, though a looper is associated with a thread. It is a class used to run a message loop for a thread. Thus using the variable name of `playbackAndPreloadThread` is a bit confusing.

But if you adopt my suggestion on when to quit the looper, then we don't need this variable in `PreloadManagerWrapper`.